src/azure-cli/azure/cli/command_modules/appconfig/commands.py

```diff
@@ -81,7 +81,7 @@ def get_custom_sdk(custom_module, client_factory, table_transformer):
     with self.command_group('appconfig replica', configstore_replica_util) as g:
         g.command('list', 'list_replica')
         g.command('create', 'create_replica')
-        g.command('delete', 'delete_replica', confirmation=True)
+        g.command('delete', 'delete_replica')
```

Comment:
May I ask why to remove the `confirmation=True`? It maybe a breaking change to customers.

Reply:
We wanted to customize the confirmation message for stores with premium sku. We haven't removed it we just added it to our function seen here - https://github.com/ChristineWanjau/azure-cli/blob/6808c7cbf8fa838610096d9f7fa477b9d963c2a2/src/azure-cli/azure/cli/command_modules/appconfig/custom.py#L308C1-L308C4